pkg/crypto/tinkcrypto/primitive/hybriddh/api/ecdhes_decrypt.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2017 Google Inc.
```

Comment:
There also seems to be a substantial number of these files in this PR.

Perhaps we should have an internal folder structure similar to what we have in fabric-sdk-go for copied files.
(does anything actually need to be exposed outside the repo - we might also need to think about third_party).

Reply:
I'll push a change that includes only the Hybrid crypto files from Tink unchanged, then I'll push new PR on top of it for ECDH-ES crypto